capstone/src/main/java/com/google/univiz/api/MapsResource.java

```diff
@@ -1,15 +1,16 @@
 package com.google.univiz.api;
 
+import com.google.univiz.CollegeData;
 import java.util.List;
 
 /**
  * The MapResource interface is responsible for all data management/processing specific for the maps
  * feature of the UniViz webapp.
  */
-public interface MapResource {
+public interface MapsResource {
   /**
    * Takes a list of college ids and returns a list of MapsData objects to the caller. This object
    * contains all data relevant to the Maps feature.
    */
-  List<MapsData> getMapData(List<CollegeId> colleges);
+  List<MapsData> getMapData(List<CollegeData> colleges);
```

Comment:
See [the opinion](https://github.com/googleinterns/univiz/pull/61#discussion_r468891190)  Chris has on passing `List<CollegeData>` in.